src/pages/index.js

```diff
@@ -99,19 +109,19 @@ class Index extends React.Component {
             <ul className="statistics">
               <li className="style1">
                 <span className="icon fa-code-fork"></span>
-                <strong>4</strong> meetups
+                <strong>6</strong> meetups
               </li>
               <li className="style3">
                 <span className="icon fa-signal"></span>
-                <strong>7,931</strong> members
+                <strong>12,000+</strong> members
```

Comment:
Don't actually do this if you've stepped away or something,

but we could just make `NUM_MEMBERS` a variable if you want the two to match easily ;)